docs/Installation.md

```diff
@@ -7,6 +7,12 @@ post-install sanity checking for Zonemaster Web GUI. The final section wraps up
 with a few pointer to other interfaces to Zonemaster. For an overview of the
 Zonemaster product, please see the [main Zonemaster Repository].
 
+The release bundle available in the Github release expects Zonemaster Web GUI
+to be served on its own domain, with the base url `/`. If you wish to serve the
+GUI on a different base url, e.g. `/zonemaster/` use
+[the instructions][custom base install] at the end of this document to build
+a custom bundle.
```

Comment:
This section is unneeded here, and creates distraction from a simple installation. If anything should be in the start of the document, it should be a table of contents as there is for e.g. Backend installation.